geniza/corpus/solr_queryset.py

```diff
@@ -44,8 +46,23 @@ def admin_search(self, search_term):
 
     keyword_search_qf = "{!type=edismax qf=$keyword_qf pf=$keyword_pf v=$keyword_query}"
 
+    # regex to convert field aliases to actual solr fields
```

Comment:
NIT: Maybe because I haven't had caffeine yet, but this took me way too long to figure out what this would look like, could we maybe just add a comment that shows what this string looks like? 
```suggestion
    # regex to convert field aliases to actual solr fields: "(shelfmark|tags| ...)"
```